### DISCUSSION TITLE
Struct ejercicio de tiempo

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#204] [Preguntas y respuestas] [answered]
Hola, esto no está bien, pero lo comparto para mostrar lo que hice. No entiendo como se relacionan las horas, minutos y segundos en una estructura. Porque se supone que tiene que haber una condición que permita que se mantenga en una base sexagecimal. ¿Se hace con una función? ¿Es cuestión de sintaxis en una estructura?  

![struct tiempo](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/7c391e3f-b816-4c76-afc1-8b351878a194)

**Comment by martinvilu** [comment, marked as answer]
¿Por qué tres `tiempo_t`? (en la función de imprimir)

Falta la primera función en la consigna, la de `crear`. Sin código no hay forma de que los números sepan que se tienen que mantener dentro de los valores establecidos.

**Reply by Clirussa** [reply]
Ok, lo reviso, gracias.

**Reply by martinvilu** [reply]
En el `printf`, podés usar `%02d`, para que se muestre como 00

Vista previa en http://zhxnlai.github.io/printf/#/
